js/backup-manager.js

```
@@ -262,6 +226,99 @@
+        // --- XP Export ---
+        if(window.getXPData) data.xp = window.getXPData();
+        // --- End XP Export ---
+
+        // Persist Discord webhook (now using IndexedDB)
+        if(window.databaseManager) {
+            // This will be handled by the database manager
+            // We don't need to include it in character data anymore
+        }
+
+        // Persist locked state
+        data.locked = (window.LockManager && window.LockManager.isLocked) ? window.LockManager.isLocked() : false;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)